pages.es/linux/waypipe.md

```diff
@@ -3,10 +3,10 @@
 > Ejecuta remotamente aplicaciones gráficas bajo un compositor para Wayland.
 > Más información: <https://gitlab.freedesktop.org/mstoeckl/waypipe>.
 
-- Ejecuta un programa gráfico remotamente y lo muestra localmente:
+- Ejecuta un programa gráfico remotamente y muestralo localmente:
```

Comment:
lo muestra localmente:

Reply:
I disagree with this one, it's easier just to place `lo` after the verb

Reply:
Do not respect the spanish semantic. by the way `muestralo` is in imperative way